Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -639,8 +652,23 @@ extension CarPlayManager {
             previewText = customPreviewText
         }
         
+        previewMapTemplate.backButton = defaultTripPreviewBackButton()
         previewMapTemplate.showTripPreviews([modifiedTrip], textConfiguration: previewText)
-        interfaceController.pushTemplate(previewMapTemplate, animated: true)
+        
+        if currentActivity == .previewing {
+            interfaceController.popTemplate(animated: false)
+            interfaceController.pushTemplate(previewMapTemplate, animated: false)
```

Comment:
Because it's already in preview mode in CarPlay, if we make the `animated: true`, we could see the preview template disappear and then pop up again. If the user on the mobile just choose an alternative route on mobile, the CarPlay may only need to refresh its preview template instead of disappear one and pop up a new one.